sklearn/datasets/_kddcup99.py

```diff
@@ -303,8 +326,7 @@ def _fetch_brute_kddcup99(data_home=None,
     except NameError:
         X = joblib.load(samples_path)
         y = joblib.load(targets_path)
-
-    return Bunch(data=X, target=y)
+    return Bunch(data=X, target=y), feat_cols, target_col
```

Comment:
`feature_names` and `target_names` would be better names

Reply:
DONE.